recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "3.0.2" %}
+{% set version = "3.0.4" %}
```

Comment:
Do we want to publish 3.0.3 first or would we rather skip that one?

Reply:
Skip. It has a bug. The only library depending on nvcomp today is libcudf, I believe, and we won’t be able to use 3.0.3.